### ISSUE TITLE
Failed to index because of descreasing version but the version did not decrease

### ISSUE THREAD

**Issue by mjegh** [andk/pause#230]
This is likely not an issue but I thought I should at least mention it.
In DBD::ODBC 1.52 the META.yml did not have a "provides".
As part of the pull request challenge I merged a change in good faith which added the "provides" but non of the entries included a version.
When I finally uploaded 1.54 including the provides I got an index error saying "Status: Decreasing version number" and "Not indexed because DBD-ODBC-1.52/ODBC.pm in M/MJ/MJEVAN/DBD-ODBC-1.52.tar.gz has a higher version number (1.52)". This error really confused me and someone else had to point out the problem which seems to be if you don't use provides it extracts the version from the modules but if you do add provides and miss off the version it assumes version 
[index_error.txt](https://github.com/andk/pause/files/513240/index_error.txt)
[META.yml.txt](https://github.com/andk/pause/files/513243/META.yml.txt)

0.

**Comment by Grinnz** [comment]
This is correct behavior; if provides metadata is provided, it must include a version if the module has one:

https://metacpan.org/pod/CPAN::Meta::Spec#provides
